src/upcoming-components/Drawer/Drawer.tsx

```diff
@@ -216,6 +215,8 @@ export const Drawer = ({
           onClick={handleOnClickOutside}
           onKeyDown={handleOnKeyDown}
           ref={ref}
+          role="button"
+          tabIndex={0}
```

Comment:
I don't think I want `role="button"` here either because the drawer will probably have buttons inside. How about we do the same thing as before and just suppress the interaction with a comment? For the `tabIndex`, I tested the component in storybook without it at all, and I actually thought it worked better. So I would just remove both lines and add the comments.

Reply:
have updated, and  added comment also